app/src/main/java/atlant/moviesapp/activity/SeasonsActivity.java

```diff
@@ -0,0 +1,147 @@
+package atlant.moviesapp.activity;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.util.Log;
+import android.view.View;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import atlant.moviesapp.R;
+import atlant.moviesapp.adapter.EpisodeAdapter;
+import atlant.moviesapp.adapter.HorizontalAdapter;
+import atlant.moviesapp.adapter.NewsFeedAdapter;
+import atlant.moviesapp.model.Episode;
+import atlant.moviesapp.presenters.SeasonsPresenter;
+import atlant.moviesapp.presenters.TvDetailsPresenter;
+import atlant.moviesapp.views.SeasonsView;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class SeasonsActivity extends AppCompatActivity implements SeasonsView {
+
+    SeasonsPresenter presenter;
+
+    @BindView(R.id.seasons_recycler_view)
+    RecyclerView seasonRecyclerView;
+
+    @BindView(R.id.episodes_recycler_view)
+    RecyclerView episodesRecyclerView;
+
+    @BindView(R.id.toolbarClassic)
+    Toolbar toolbar;
+
+    @BindView(R.id.season_year)
+    TextView seasonYear;
+
+    private Integer seriesId, seasonId, seasonNum;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_seasons);
+
+        ButterKnife.bind(this);
+
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setTitle(R.string.tvshows_title);
+
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setDisplayShowHomeEnabled(true);
+        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                onBackPressed();
+            }
+        });
+
+        Intent intent = getIntent();
+        seriesId = intent.getIntExtra("showId", 0);
```

Comment:
Put all String, constants, keys used to string resources. If they are keys, make translatable false, and if they are used words, sentences that are visible and shown to the user do not put the translatable attr.